locales/en-US/app.l20n

```diff
@@ -104,8 +103,10 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainerIntro "There are">
+<userCountContainerOutro "people trying {{title}} right now!">
```

Comment:
I think i'd just need to pass the span in, b/c The user count comes from the model